packages/markdown/processor/compile/index.js

```diff
@@ -4,3 +4,4 @@ export { default as rdmeEmbedCompiler } from './embed';
 export { default as rdmeVarCompiler } from './var';
 export { default as rdmeCalloutCompiler } from './callout';
 export { default as rdmePinCompiler } from './pin';
+export { default as plainTextCompiler } from './plain-text';
```

Comment:
I think of this folder as being exclusively for methods that modify the actual Unified `Compiler` instance, like so:

https://github.com/readmeio/api-explorer/blob/66eeca17c49cf30d27cbf1277ea7118bdac9a91a/packages/markdown/processor/compile/callout.js#L3

Can we move this `plain-text.js` script to the `plugin/` directory? Should work just as well from there!

Reply:
Happy to move it.